deployment/terraform/batch.tf

```diff
@@ -62,6 +62,7 @@ data "template_file" "default_job_definition" {
     environment                = "${var.environment}"
     django_secret_key          = "${var.django_secret_key}"
     google_server_side_api_key = "${var.google_server_side_api_key}"
+    google_analytics_key       = "${var.google_analytics_key}"
```

Comment:
This isn't being used as it isn't within the job definition JSON file.

I think we can remove this altogether though, since we're not passing any other client side environment variables to the job definition.

Reply:
Done in the latest version. Also removed `rollbar_client_side_access_token` from this file.